apps/shopify/src/dataTransformer.js

```diff
@@ -9,12 +9,14 @@ import flatten from 'lodash/flatten';
 export const dataTransformer = product => {
   const image = get(product, ['image', 'src'], '');
   const sku = get(product, ['sku'], '');
+  const variantSku = get(product, ['variantSKU'], '');
 
   return {
     id: product.id,
     image,
     name: product.title,
-    sku
+    readableSKU: variantSku,
```

Comment:
I wonder, do we care about consistency with capitalization when naming vars? e.g. both would be `readableSku` and `variantSku` or `readableSKU` and `variantSKU`.

Reply:
Yeah, good point!